src/content/docs/en/reference/experimental-flags/responsive-images.mdx

```diff
@@ -33,22 +41,37 @@ Additionally, Astro's image components can receive [responsive image props](#res
 
 Images in your `public/` folder are never optimized, and responsive images are not supported.
 
-## Responsive image configuration settings
+:::note
+Enabling responsive images will generate additional image sizes for all affected images. For prerendered pages this happens during the build so may increase the build time of your project, especially if you have a large number of images. For pages rendered on-demand the images are generated as-needed, so this has no impact on build times but may increase the number of transformations performed. Depending on your image service this may incur additional costs. 
+:::
+
+## Image layout
+
+In order to generate the correct `srcset` and `sizes` attributes, the `<Image />` and `<Picture />` components need to know how the image should resize when its container changes size. This is done by setting the `layout` prop, or `image.experimentalLayout` default. The supported values are:
+
+- `constrained` - The image will scale to fit the container, maintaining its aspect ratio, but will not exceed the specified `width` and `height`, or the image's original dimensions.
+- `full-width` - The image will scale to fit the width of the container, maintaining its aspect ratio. Use this for hero images or other images that should take up the full width of the page.
+- `fixed` - The image will maintain the requested dimensions and not resize. The `srcset` will be generated for the requested image size, plus 2x resolution for high-density displays if the source image is large enough.
```

Comment:
```suggestion
- `fixed` - The image will maintain the requested dimensions and not resize. The `srcset` will be generated for the requested image size, plus 2x resolution for high-density displays if the source image is large enough. Use this when you prefer to see your image cropped at smaller sizes instead of showing a smaller version of the whole picture.
```
OK, I don't love this, but I do think an explanation of the container making only part of the image visible, at its original size, is helpful here for the Use when.../Useful for...

Reply:
I clearly need to explain this better, because this isn't what it does! It just displays them with no resizing at all, like a normal img tag with no styling.

Reply:
I've rewritten this to try and make it clearer. lmk if it makes sense!